public/style.css

```diff
@@ -14,4 +25,31 @@
 
 ul {
   list-style: none;
+  display: grid;
+  grid-template-columns: repeat(2, 2fr);
+  row-gap: 1rem;
+}
+
+.answer-list-item input[type='radio'] {
+  opacity: 0;
+  position: fixed;
+  width: 0;
+}
+
+.answer-list-item label {
+  display: inline-block;
+  background-color: #010508;
+  color: white;
+  padding: 12px 24px;
+  clip-path: polygon(10% 0, 90% 0, 100% 50%, 90% 100%, 10% 100%, 0% 50%);
```

Comment:
I have a tip:
I have a hard time to imagine what this button with clip-path looks like.
When you make a PR with 'difficult' styling, or just a lot of styling, I usually add one of more screenshots to my PR.